features/machine/machine.feature

```diff
@@ -99,6 +100,7 @@ Feature: Machine features testing
       | type          | merge                                  |
       | n             | openshift-machine-api                  |
     Then the step should succeed
+    Given 10 seconds have passed
```

Comment:
any more reliable way to wait?

Reply:
@jhou1 how about wrap about the subsequent steps with `And I wait up to xxx seconds for the steps to pass:` to bypass the hardcoded wait?

Reply:
or just use `Given I wait for the steps to pass:` which defaults to 60 seconds...

Reply:
Done, thanks